Dockerfile.tarball

```diff
@@ -0,0 +1,31 @@
+FROM cilium/tetragon:latest as tetragon-release
+
+RUN mkdir -p /etc/tetragon/tetragon.conf.d/
+
```

Comment:
I saw all the config stuff is placed under /usr/local/lib/tetragon/tetragon.conf.d
do we need to keep both directories?

Reply:
The /etc/tetragon/ during install will be empty! hmm just to make it easy for users but no strong opinion on this

Reply:
The logic is /usr/ for us and packages and /etc for administrators: https://github.com/cilium/tetragon/pull/635